code/modules/antagonists/gang/gang.dm

```diff
@@ -67,6 +65,8 @@
 /datum/antagonist/gang/on_gain()
 	if(starter_gangster)
 		equip_gangster_in_inventory()
+	var/datum/atom_hud/H = GLOB.huds[ANTAG_HUD_GANGSTER]
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names